.docs/summary-generator.cjs

```diff
@@ -1,14 +1,13 @@
 const path = require("path");
 const fs = require("fs");
-const docsDir = "docs";
-const directoryPath = path.join(__dirname, "..", docsDir);
 
-async function prepareDocAnchor(type, file) {
+async function prepareDocAnchor(docsDir, type, file) {
+  const directoryPath = path.join(docsDir);
```

Comment:
this is probably redundant. It could be: `const filePath = path.join(docsDir, type, file);`